api/python/quilt3/api.py

```diff
@@ -128,58 +128,23 @@ def delete_package(name, registry=None):
         registry (str): The registry the package will be removed from
     """
     validate_package_name(name)
-
-    if name not in list_packages(registry):
-        raise QuiltException("No such package exists in the given directory.")
+    usr, pkg = name.split('/')
 
     registry_base_path = get_package_registry(fix_url(registry) if registry else None)
-    registry_url = urlparse(registry_base_path)
-    pkg_namespace, pkg_subname = name.split("/")
-
-    tophashes_with_packages = _tophashes_with_packages(registry)
-
-    if registry_url.scheme == 'file':
-
```

Comment:
Is this removing the delete code for local/file-system package stores. Is that intentional? I wouldn’t think we need to remove that to improve the S3 code performance.

Reply:
You mean, the cleanup code? I wanted it to be consistent between local and S3, so it'll do the same thing - delete the package versions, but not the actual manifests. We can add a cleanup helper later - or hopefully, just migrate to a better directory structure.

Reply:
OK, that makes sense. I couldn't tell from just the diff if users could still call delete on local packages.